ts/input/mathml/MathMLCompile.ts

```diff
@@ -174,18 +174,19 @@ export class MathMLCompile<N, T, D> {
         if (mml.arity === 0) {
             return;
         }
-        for (const child of this.adaptor.childNodes((node)) as N[]) {
-            const name = this.adaptor.kind(child);
+        const adaptor = this.adaptor;
+        for (const child of adaptor.childNodes((node)) as N[]) {
```

Comment:
Is there a reason for the double parenthesised `((node))`?

Reply:
I think they are probably left over from some older `as N` cast or some such thing.  I'll remove them.